system_tests/storage.py

```diff
@@ -1,4 +1,4 @@
-# Copyright 2014 Google Inc.
+# Copyright 2016 Google Inc.
```

Comment:
I don't think this is correct:  the "main" copyright is for the year the file was created, not the last year in which it was updated.